lib_test/jbuild

```diff
@@ -1,9 +1,13 @@
 (executable
  ((name suite)
-  (libraries (lwt.unix nbd oUnit))
+  (libraries
+   (alcotest
+    io-page-unix
+    lwt.unix
+    nbd))
   ))
 
 (alias
  ((name   runtest)
   (deps   (suite.exe))
-  (action (run ${<} -runner sequential))))
+  (action (run ${<}))))
```

Comment:
Because that was a parameter for ounit, but now we switched to alcotest, which works fine by default :)

Reply:
Alcotest automatically instruments bisect?

Reply:
What I meant is that we don't need the ` -runner sequential` parameter, because that was for jbuilder.

Reply:
I ma not sure what happened, I saw this comment on a commit removing the bisect flags :confused:

Reply:
Oh, you mean this one: https://github.com/xapi-project/nbd/pull/118/commits/b6b3b862633138497fcf7d0687faf43dd09955bc ? I decided not to include test code in coverage analysis , because that changes the coverage significantly, and we're interested in the main code.

Reply:
I agree with this decision